packages/tab-list/src/tab-list.ts

```diff
@@ -75,6 +83,15 @@ export class TabList extends Focusable {
             this.querySelector('sp-tab')) as Tab;
     }
 
+    constructor() {
+        super()
+
+        // These can be added as @click and @keydown handlers on the
+        // slot once we no longer need web component polyfills
+        this.addEventListener('click', this.onClick);
+        this.addEventListener('keydown', this.onKeyDown);
```

Comment:
Instead of trying to add these to the slot, I realised I can just add them to the element itself and then I don't have to worry about cleanup. You may have been trying to tell me that 🤦